apps/services/sessions/src/app/cleanup/worker.service.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { createNationalId } from '@island.is/testing/fixtures'
+import { TestApp, testServer, useDatabase } from '@island.is/testing/nest'
+
+import { FixtureFactory } from '../../../test/fixture.factory'
+import { SequelizeConfigService } from '../../sequelizeConfig.service'
+import { Session } from '../sessions/session.model'
+import { SessionsCleanupWorkerModule } from './worker.module'
+import { SessionsCleanupService } from './worker.service'
+
+const setupWithoutAuth = async (): Promise<TestApp> =>
```

Comment:
Do you know about the `libs/testing/nest/src/lib/setup.ts` shared utility for simple reuse of `setupApp` and `setupAppWithoutAuth`?
Should be able to just use that here.

Reply:
Fixed.